src/components/views/about-us/where/where.component.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./where.module.scss";
+import backgroundImage from "public/assets/images/aboutUs/Map.png";
+import Image from "next/image";
+// import { useTranslation } from "next-i18next";
+import { Container } from "@design-system/layout/utilities";
+
+function Where() {
+  // const { t } = useTranslation("about-us");
```

Comment:
zamiast wykomentowywać nieużywanego kodu po prostu go usuń. Opróćz tego nazwij to ten plik sekcje np "LocalizationSection" zamiast "Where"